Building the the library fails

After check out the repository, i used the instructions to build the minified files with yarn and npm:
`npm run build`  / `yarn build` throws the following Error:
Error: Cannot find module 'SvgPath' from '/home/fabbud/svg2pdf.js/src' at /home/fabbud/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17

execute `yarn browserify:svgpath` / ` npm run browserify:svgpath` initialize the SvgPath 
$ browserify node_modules/svgpath/index.js -s SvgPath -o src/SvgPath.js
Done in 1.20s.
then 
`yarn run browserify`  or `yarn build` will give the same above error.

yarn version v1.16.0

```sh
node node_modules/browserify/bin/cmd.js node_modules/svgpath/index.js -s SvgPath -o src/SvgPath.js
```
works, but the next one fails

```sh
node node_modules/browserify/bin/cmd.js src/svg2pdf.js --debug -p licensify -s svg2pdf -o dist/svg2pdf.js

Error: Cannot find module 'SvgPath' from '/home/ber/svg2pdf.js/src'
    at /home/ber/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/home/ber/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/home/ber/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/home/ber/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/ber/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/ber/svg2pdf.js/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:154:21)
```

--- a/src/svg2pdf.js
+++ b/src/svg2pdf.js
@@ -2316,7 +2316,7 @@ SOFTWARE.
   };
 
   if (typeof define === "function" && define.amd) {
-    define(["./rgbcolor", "SvgPath", "font-family-papandreou", "cssesc"], function (rgbcolor, svgpath, fontFamily, cssesc) {
+    define(["./rgbcolor", "svgpath", "font-family-papandreou", "cssesc"], function (rgbcolor, svgpath, fontFamily, cssesc) {
       RGBColor = rgbcolor;
       SvgPath = svgpath;
       FontFamily = fontFamily;

--- a/src/svg2pdf.js
+++ b/src/svg2pdf.js
@@ -2325,7 +2325,7 @@ SOFTWARE.
     });
   } else if (typeof module !== "undefined" && module.exports) {
     RGBColor = require("./rgbcolor.js");
-    SvgPath = require("SvgPath");
+    SvgPath = require("svgpath");
     FontFamily = require("font-family-papandreou");
     cssEsc = require("cssesc");
     module.exports = svg2pdf;

